presentation/common/src/presentation-common/PresentationIpcInterface.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Core
+ */
+
+import { RulesetVariableJSON } from "./RulesetVariables";
+
+/** @internal */
+export const presentationIpcChannel = "presentationIpcInterface-1.0";
+

```

Comment:
Not a fan of this naming, the following looks much better to me:
```ts
export const PRESENTATION_IPC_CHANNEL_NAME = "PresentationIpcInterface-1.0";
```
Is your version based on some pattern in core?

Reply:
Changed variable name. I am not sure if there is a pattern for version. As I know version in channel name is optional, so I added this just for future if we want to change channel name for some reason. I can use different format or remove version from name if you want.

Reply:
"version" in my comment meant your code version, not the version of the IPC interface. But now that you mention it, I'm not sure we need to have version in the channel name - in IPC apps backends and frontends are tightly coupled, so there should be no version incompatibilities, right? If that's the case, I'd say just change it to `"PresentationIpcInterface"`.

Reply:
Oh... Yes I based on documentation: https://github.com/imodeljs/imodeljs/blob/master/docs/learning/IpcInterface.md#creating-your-own-ipc-interfaces-and-ipchandlers

I think we can remove version.

Reply:
Agree.

Reply:
Also I think the name should either start with upper case letter, or be in a format "presentation-ipc-interface".

Reply:
Removed version and changed to "presentation-ipc-interface"